src/Maker/MakeCommand.php

```diff
@@ -63,6 +64,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             [
                 'command_name' => $commandName,
                 'set_description' => !class_exists(LazyCommand::class),
+                'use_command_attribute' => 80000 <= \PHP_VERSION_ID && class_exists(AsCommand::class),
```

Comment:
Under a different PR, I think we can add a helper to the `PhpCompatUtil` to eliminate having to check the PHP version before calling `class_exists()`. But the complexity required to make that happen is beyond the scope of this PR.

https://bugs.php.net/bug.php?id=80938 explains why this must be done like this.